azure-quantum/azure/quantum/workspace.py

```diff
@@ -409,23 +418,45 @@ def refresh_session(
         self,
         session: Session
     ):
+        """Updates the session details with the latest information
+           from the workspace.
+
+        :param session: The session to be refreshed.
+        :type session: Session
+        """
         session.details = self.get_session(session_id=session.id).details
 
     def get_session(
         self,
-        session_id:str
+        session_id: str
```

Comment:
Ideally the session method would just take id as the param instead of session_id. But I see that the job methods take job_id so may be we keep it as is for consistency.

Reply:
Yes, that's why I kept it like this. Also it makes it clear that this is the session's id (since we are in the workspace object).
The session object does not have the `session_` prefix.